sfa_dash/templates/index.html

```diff
@@ -6,33 +6,58 @@
 <h2 class="page-title">Solar Forecast Arbiter Dashboard</h2>
 {% include "sections/notifications.html" %}
 {% if current_user is none %}
-<p>Welcome to the Solar Forecast Arbiter Dashboard. If you have an account, please
+<p>Welcome to the Solar Forecast Arbiter, a tool for evaluating your
+solar power and irradiance forecasts. The Arbiter will allow you to securely
+upload forecast data for your own private evaluation, or to participate in
+competitive forecast trials that compare forecast performance between
+anonymous participants.</p>
```

Comment:
can you edit and html-ify the following?

The Arbiter allows you to:

* Securely upload forecast and observation data
* Analyze historical forecast accuracy and track ongoing forecast accuracy
* Optionally share data and analysis reports with other users
* Participate in anonymous, multi-vendor, operational forecast trials